### PR TITLE
Fixes #757 Added Queens Problem

### LINKED ISSUE #757: N-Queens Problem
I would like to add N-Queens Problem that uses backtracking technique, in the following languages:
- C/C++
- Java
- Python

### PR DESCRIPTION
**Pull request by marwahmanbir**
Adding Queens Problem algorithm in:
- C
- Java
- Python

Fixed #757

### CODE REVIEW

**Comment by marwahmanbir** [comment]
@jainaman224 Can you please review this PR?

**Comment by marwahmanbir** [comment]
Force pushed so as to drop the last two commits to keep the commit history clean.

**Comment by marwahmanbir** [comment]
Added a single new line at the end of file.
Added sample input output at the end of file.

File: Queens_Problem/Queens_Problem.c [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    //calling the function
+    N_queen(N);
+    //printing the matix
+    for(i=0;i<N;i++)
+    {
+        for(j=0;j<N;j++)
+            printf("%d\t",board[i][j]);
+        printf("\n");
+    }
+
```
Comment (GOVINDDIXIT) [inline Queens_Problem/Queens_Problem.c]: Extra

File: Queens_Problem/Queens_Problem.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        //calling the function
+        nQueen(N);
+        //printing the matix
+        for(i=0;i<N;i++)
+        {
+            for(j=0;j<N;j++)
+                System.out.print(board[i][j]+"\t");
+            System.out.print("\n");
+        }
+
```
Comment (GOVINDDIXIT) [inline Queens_Problem/Queens_Problem.java]: Extra

**Comment by marwahmanbir** [comment]
@GOVINDDIXIT I was specifically told to add an extra line at the end of every file by @jainaman224. You can see the requested change above my previous comment and also the commit I made specially to add a single newline at the end of every doc.

**Comment by GOVINDDIXIT** [comment]
I am talking about extra line in between the code. To be more specific Line 87 of Queens_Problem.c and  line 81 of Queens_Problem.java

**Comment by marwahmanbir** [comment]
Oh, sure, I'll remove them.

**Comment by marwahmanbir** [comment]
Done.

**Comment by marwahmanbir** [comment]
@GOVINDDIXIT @jainaman224 Can you please review it? I've made the mentioned changes.

**Comment by marwahmanbir** [comment]
@GOVINDDIXIT Updated.

**Pull request merged**